README.md

```diff
@@ -105,102 +133,87 @@ the generated headers correctly. The C++ bindings currently require C++17. If
 you discover a compatibility problem with a generated pipeline, please open an
 issue.
 
-# Building Halide with Make
-
-### TL;DR
-
-Have llvm-16.0 (or greater) installed and run `make` in the root directory of
-the repository (where this README is).
-
-### Acquiring LLVM
+## Acquiring LLVM
 
 At any point in time, building Halide requires either the latest stable version
 of LLVM, the previous stable version of LLVM, and trunk. At the time of writing,
-this means versions 18, 17, and 16 are supported, but 15 is not. The commands
-`llvm-config` and `clang` must be somewhere in the path.
+this means versions 19, 18, and 17 are supported, but 16 is not.
 
-If your OS does not have packages for LLVM, you can find binaries for it at
-http://llvm.org/releases/download.html. Download an appropriate package and then
-either install it, or at least put the `bin` subdirectory in your path. (This
-works well on OS X and Ubuntu.)
+It is easy to get a binary release of LLVM on macOS by using
```

Comment:
nit: easy -> simplest

brew can get into a weird state fairly easily, and you don't want to make the reader feel dumb if they encounter errors doing something that was supposed to be easy